src/Parameter.hpp

```diff
@@ -37,7 +41,9 @@ namespace imu_aceinna_openimu {
         {"point-of-interest-x", 17, PARAM_DOUBLE, nullptr},
         {"point-of-interest-y", 18, PARAM_DOUBLE, nullptr},
         {"point-of-interest-z", 19, PARAM_DOUBLE, nullptr},
-        {"rtk-heading-to-mag-heading", 20, PARAM_DOUBLE, "DEGREES"},
+        {"rtk-heading-to-mag-heading",
+         static_cast<int>(FirmwareParameterIndex::RTK_HEADING_TO_MAG_HEADING),
+         PARAM_DOUBLE, "DEGREES"},
```

Comment:
Should be radians.

Reply:
The conversion is done at firmware side https://github.com/tidewise/firmwares-imu_aceinna_openimu/blob/97956eaca04bdd4eefb448a13ff9309caa9cc789/src/user/UserConfiguration.c#L250, since it is a user config I found it more readable to use degrees

Reply:
It is only confusing. No user will write the binary message that has to be sent to the IMU for configuration.

All code interfaces must use radians. Period. No exceptions.

The tooling (imu_aceinna_...), however, can do the conversion if you want it to.

Reply:
> The tooling (imu_aceinna_...), however, can do the conversion if you want it to.

Yes, I only perceived that later down the road. Chaging it in other PR